CI-CD/AzureDevOps/azure-pipelines_dotnet_sbom.yml

```diff
@@ -57,6 +57,7 @@ steps:
   env:
     WS_APIKEY: $(APIKEY)
     WS_USERKEY: $(USERKEY)
+    WS_FILESYSTEMSCAN: false
```

Comment:
Wrong place, this task is just the script that retrieves the risk report, the fileSystemScan parameter belongs in the UA scan

Reply:
Fixed